themes/default/layouts/page/pricing.html

```diff
@@ -1012,124 +496,679 @@ <h5>Sample Email</h5>
-                        trial expires, no data will be lost, and there is a grace period with soft enforcement.
+                        You can keep track of current credit usage and upcoming charges by navigating to
+                        Settings and then Billing & Usage in the Pulumi console.
                     </p>
                 </div>
             </li>
 
-            <li class="accordion-item text-2xl py-3 border-t-2">
-                {{ partial "accordian-header" (dict "text" "How can organizations not seeking a profit use Pulumi for free?" "large_header" true) }}
+            <li class="accordion-item text-2xl py-3">
+                {{ partial "accordian-header" (dict "text" "When will I be billed for using the Team Edition?" "large_header" true) }}
 
                 <div class="accordion-item-body-no-animation text-base">
                     <p>
-                        The <a class="link" href="{{ relref . "open-source-free-tier/" }}">Open-Source Free Tier</a> allows organizations not seeking a profit with projects under an open-source license to use Pulumi for free.
+                        You will be billed for the previous month’s usage on the first day of each month.
                     </p>
                 </div>
             </li>
 
-            <li class="accordion-item text-2xl py-3 border-t-2">
-                {{ partial "accordian-header" (dict "text" "How much will Pulumi cost?" "large_header" true) }}
+            <li class="accordion-item text-2xl py-3">
+                {{ partial "accordian-header" (dict "text" "How can groups not seeking a profit use Pulumi for free?" "large_header" true) }}
 
                 <div class="accordion-item-body-no-animation text-base">
                     <p>
-                        The Team Starter edition of Pulumi has a single price
-                        that includes 3 users. You cannot extend your account beyond these 3 users.
-                    </p>
-                    <p>
-                        The Team Pro and Enterprise editions are priced per user, with a minimum number of
-                        users per organization. Team Pro offers up to 25 team members, while Enterprise
-                        offers unlimited number of seats in your organization.
-                    </p>
-                    <p>
-                        All editions are available as annual subscriptions. You may purchase monthly
-                        subscriptions for an additional 20% per month.
-                    </p>
-
-                    <p>
-                        For large numbers of users and custom pricing, support options, on-premises, or to
-                        discuss training and services options, please <a class="link" href="#contact">contact us</a>.
+                        The <a class="underline" href="{{ relref . "open-source-free-tier/" }}">Open-Source Free Tier</a> allows
```

Comment:
```suggestion
                        The <a class="underline" href="{{ relref . "open-source-free-tier/" }}">Open-Source Free Edition</a> allows
```

Reply:
i do no think we should change this, it is the heading on the page: https://www.pulumi.com/pricing/open-source-free-tier/